themes/tiddlywiki/vanilla/sidebar-resizer.tid

```diff
@@ -0,0 +1,158 @@
+title: $:/themes/tiddlywiki/vanilla/sidebar-resizer
+tags: [[$:/tags/Stylesheet]]
+code-body: yes
+
+\define sidebarbreakpoint()
+<$text text={{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}}/>
+\end
+
+\define sidebarbreakpoint-minus-one()
+<$text text={{{ [{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}removesuffix[px]subtract[1]addsuffix[px]] ~[{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}] }}}/>
+\end
+
+\rules only filteredtranscludeinline transcludeinline macrodef macrocallinline macrocallblock conditional
+
+@media (min-width: <<sidebarbreakpoint>>) {
+
+	.tc-sidebar-resizer {
+		position: fixed;
+		top: 0;
+		right: calc({{$:/themes/tiddlywiki/vanilla/metrics/sidebarwidth}} - 42px);
+		height: 100vh;
+		width: 22px;
+		z-index: 1;
+		transition: opacity 100ms;
+		opacity: 0.1;
+		cursor: ew-resize;
+		background: linear-gradient(<<colour muted-foreground>>, <<colour muted-foreground>>) no-repeat center/2px 100%;
+		touch-action: none;
+		user-select: none;
+	}
+	
+	.tc-sidebar-resizer-pointerdown-eventcatcher {
```

Comment:
I think all the initialization settings that are the same for all elements could be in 1 rule, with

```
.tc-sidebar-resizer-pointerdown-eventcatcher,
.tc-sidebar-resizer-pointermove-eventcatcher,
.tc-sidebar-resizer-pointermove {
... /* settings that are the same for all of them */
}
```
```

Reply:
Yep, will do!

Reply:
Done